service/config/index.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+// Configuration : Struct to contain all configuration
+type Configuration struct {
+	KeywordPriority []string `json:"keywordPriority"`
+	UserAgents []string `json:"userAgents"`
+}
+
+var config Configuration
```

Comment:
remove this and make `New()` return `&Configuration{}` and then have another function `func (c *Configuration) Load() {...} *Configuration` that loads the configuration.

Reply:
So what I have understood is, everywhere we want to access the configuration we would need to call `New()` to create the object and `Load()` to load the config data into the object. 

If I have understood correctly, this will load the `config.json` on every request and slow down the response time.

Reply:
Yaa.. it's somehow correct. We were planning to load it once only at the starting, so you don't need to load it in every request. This was only to maintain everything in a modular way, so will ease the development and make the code look clean later.